app/components/sidebars/main/channels_list/channel_item/channel_item.test.js

```diff
@@ -41,7 +41,7 @@ describe('ChannelItem', () => {
         isBot: false,
         customStatusEnabled: true,
         channelHasCall: false,
-        callsFeatureEnabled: false,
+        isSupportedServerCalls: false,
```

Comment:
I don't see this prop in the component

Reply:
Ah, good catch -- I discovered I could remove the `isSupportedServerCalls` from the `ChannelItem` and forgot to remove it here. This removes the need for a test, so I deleted it (the base props test is the 'no calls' state).